release_notes/RELEASE.txt

```diff
@@ -25,7 +25,6 @@ CONTENTS
 - New features and changes
   -- Configuration
   -- C Library
-- Support for new platforms and compilers
 - Bugs fixed since HDF 4.2.16
```

Comment:
should be 4.2.16-2 as the one below

Reply:
Hmmm.  There's nothing in the HISTORY.txt file for a 4.2.16-2 release.  I don't remember why there was a 4.2.16-2 patch release, but I wonder if the previous versions in this RELEASE.txt should all be 4.2.16?
   
There are 2 instances of 4.2.16 and one of 4.2.16-2, so I suppose all 3 in this file should be the same.

Reply:
The 4.3.0 release should be off the 4.2.16, IIRC. A patch realease technically isn't a real release.